examples/apps/collaborative-textarea/README.md

```diff
@@ -7,7 +7,7 @@ component to launch a basic collaborative HTML `<textarea>`
 
 If you want to run this example follow the following steps:
 
-1. Run `npm install` from the `FluidFramework` root directory
+1. Run `npm install` and `npm run build:fast` from the `FluidFramework` root directory
```

Comment:
nit: To build _only_ what's needed for the example, you could use `npm run build:fast @fluid-example/collaborative-textarea`. That would build only the pre-requisites for that project, along with the project itself. I don't know if it's worth updating each one of these with the project name. Your call.